packages/common-grpc/test/service.js

```diff
@@ -21,6 +21,7 @@ var duplexify = require('duplexify');
 var extend = require('extend');
 var googleProtoFiles = require('google-proto-files');
 var grpc = require('grpc');
+var grpcVersion = require('grpc/package.json').version;
```

Comment:
Let's remove this and just make an `EXPECTED_API_CLIENT_HEADER` that we can use in the various places in the tests:

```js
var EXPECTED_API_CLIENT_HEADER = [
  'gl-node/' + process.versions.node,
  'gccl/' + CONFIG.packageJson.version,
  'grpc/' + require('grpc/package.json').version
].join(' ');
```